src/index.js

```diff
@@ -8,7 +8,9 @@ import delve from 'dlv';
  */
 export default function linkState(component, key, eventPath) {
 	let path = key.split('.');
-	return function(e) {
+	const cache = component._linkedStates || (component._linkedStates = {});
+
+	return cache[`${key}\n${eventPath}`] || (cache[`${key}\n${eventPath}`] = function(e) {
```

Comment:
why not make `${key}\n${eventPath}` to be a variable?

Reply:
That's a good suggestion and I normally would do that in my own code but in this case it increases the gzipped file size from 315 bytes to 317 bytes. I guess that's a pretty inconsequential difference but as the philosophy of preact is "as small a file size as possible" I thought that maybe the smaller file size would be preferable. What do you think?